### PR TITLE
Create citizen-science-theory-and-practice.csl

### PR DESCRIPTION
**Pull request by damnation333**
via https://forums.zotero.org/discussion/61206/style-request-citizen-science-theory-and-practice
Very nice guidelines.
Adapted for everything in there.
Not sure why in the bibliography it's only giving me 7 authors and not all of them?!

### CODE REVIEW

**Comment by adam3smith** [comment]
Do you need the et-al conditions in the author-short and editor macro? Those are somehow messing with the et-al for the bibliography. While that's a bug, may not be worth getting into. Alternatively, you can set explicit et-als for the bibliography and make them arbitrary large (99 or 999)

File: citizen-science-theory-and-practice.csl [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    <link href="http://theoryandpractice.citizenscienceassociation.org/about/submissions/#References" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="sociology"/>
+    <category field="social_science"/>
+    <category field="anthropology"/>
+    <category field="medicine"/>
```
Comment (rmzelle) [inline citizen-science-theory-and-practice.csl]: @damnation333, based on http://theoryandpractice.citizenscienceassociation.org/about/, maybe we should just limit this to the "science" category?

File: citizen-science-theory-and-practice.csl [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+          <text term="available at" text-case="capitalize-first"/>
+          <text variable="URL"/>
+          <group delimiter=" " prefix="[" suffix="]">
+            <text term="accessed" text-case="capitalize-first"/>
+            <date form="text" variable="accessed"/>
+          </group>
+        </group>
+      </if>
+      <else>
+        <text variable="DOI" prefix="DOI: http://dx.doi.org/"/>
```
Comment (rmzelle) [inline citizen-science-theory-and-practice.csl]: https://doi.org/ here please, even if the guidelines say otherwise

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**